section2-week1/exercise7/Main.cpp

```diff
@@ -11,93 +11,96 @@ using namespace std;
 
 int main()
 {
-	int value, random, answer;
-	ans = 0;
+	int value, random;
 
-	srand(time(NULL));
-	random = rand() % (1 - 4);
 
 	cout << "Hey you, let's play Rock, Paper, Scissors!" << endl;
-
 	cout << "1 - Rock\n2 - Paper\n3 - Scissors" << endl;
 	cout << "choose you weapon: " << endl;
 
-	cin >> value;
+	enum Weapon
+	{
+		Rock, Paper, Scissors
+	};
+	srand(time(NULL));
+	random = rand() % (1 - 4);
+	Weapon weapon = (Weapon)random;
 
-	cout << "\nI have selected the weapon: " << random << endl;
+	cin >> value;
```

Comment:
Same thing for the variable `value`. What is `value` supposed to mean? What about `userChoice` or `userWeapon`? Do you see how that would make the code a lot clearer?